modules/evm/rpc/src/call_request.rs

```diff
@@ -39,3 +39,16 @@ pub struct CallRequest {
 	/// Data
 	pub data: Option<Bytes>,
 }
+
+/// EstimateResources response
+#[derive(Debug, Default, PartialEq, Serialize, Deserialize, Clone)]
+#[serde(deny_unknown_fields)]
+#[serde(rename_all = "camelCase")]
+pub struct EstimateResourcesResponse {
+	/// Used gas
+	pub gas: U256,
+	/// Used storage
+	pub storage: U256,
+	/// Adjusted weight fee
+	pub weight_fee: U256,
```

Comment:
should be whatever original adjusted_weight_fee type

Reply:
The serde does not support u128, use U256 instead of u128.